### ISSUE TITLE
Crash probably in F# or Shared project

### ISSUE THREAD

**Issue by xperiandri** [DarkDaskin/VSTabPath#6]
```
SetSite failed for package [TabPathPackage]Source: 'mscorlib' Description: The path is not of a legal form. System.ArgumentException: The path is not of a legal form.
   at System.IO.Path.NormalizePath(String path, Boolean fullCheck, Int32 maxPathLength, Boolean expandShortPaths)
   at System.IO.Path.InternalGetDirectoryName(String path)
   at VSTabPath.TabTitleManager..ctor() in C:\projects\vstabpath\VSTabPath\TabTitleManager.cs:line 64
   at VSTabPath.TabTitleManager.<>c.<EnsureTabTitleManager>b__3_0() in C:\projects\vstabpath\VSTabPath\TabTitleManager.cs:line 36
   at VSTabPath.TabTitleManager.EnsurePropertyValue[T,TProperty](T target, DependencyProperty property, Func`1 factory) in C:\projects\vstabpath\VSTabPath\TabTitleManager.cs:line 119
   at VSTabPath.TabTitleManager.EnsureTabTitleManager(ViewGroup target) in C:\projects\vstabpath\VSTabPath\TabTitleManager.cs:line 36
   at VSTabPath.TabPathViewElementFactory.SetupView(DocumentView view) in C:\projects\vstabpath\VSTabPath\TabPathViewElementFactory.cs:line 40
   at VSTabPath.TabPathViewElementFactory.SetupExistingDocuments() in C:\projects\vstabpath\VSTabPath\TabPathViewElementFactory.cs:line 56
   at VSTabPath.TabPathPackage.<InitializeAsync>d__2.MoveNext() in C:\projects\vstabpath\VSTabPath\TabPathPackage.cs:line 69 --- End of stack trace from previous location where exception was thrown ---    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Shell.AsyncPackage.<>c__DisplayClass20_0.<<Microsoft-VisualStudio-Shell-Interop-IAsyncLoadablePackageInitialize-Initialize>b__1>d.MoveNext() --- End of stack trace from previous location where exception was thrown ---    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.JoinableTask.<JoinAsync>d__68.MoveNext()
```

**Comment by xperiandri** [comment]
Path to solution `C:\Users\Andrii\Dev\Secret Circle\Oil change dialog`

**Comment by DarkDaskin** [comment]
I was able to reproduce the exception by opening a project which does not belong to a solution.

Is that what you were doing?

**Comment by DarkDaskin** [comment]
Closing because of inactivity. The problem is probably fixed by def0f332cef694e17ed34172a8b69dbc07b8c963.

**Comment by bddckr** [comment]
Just to note: I'm using VSTabPath 1.3 in VS 2022 and have both C# and F# projects in the same solution and it's all working great on my end.

Thanks for the great work!
